### ISSUE TITLE
PDFs not created

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/Cer2016#87]
```
p230198@pg-login:scripts egrep -iR "error" --include=*.log
collect_gammas.log:!pdfTeX error: pdflatex (file /local/7150865/tex2pdf.15745/da39a3ee5e6b4b0d3255
collect_gammas.log: ==> Fatal error occurred, no output PDF file produced!
collect_gammas.log:pandoc: Error producing PDF
collect_nltts.log:!pdfTeX error: pdflatex (file /local/7150867/tex2pdf.15901/da39a3ee5e6b4b0d3255
collect_nltts.log: ==> Fatal error occurred, no output PDF file produced!
collect_nltts.log:pandoc: Error producing PDF
```

**Comment by richelbilderbeek** [comment]
Added `warnings()` in all R scripts. Found this:

```
Warning messages:
1: In dev.off() : No png support in this version of R
```

**Comment by richelbilderbeek** [comment]
Try a newer version of R...

**Comment by richelbilderbeek** [comment]
`sed` command used:

```
sed -i 's/3.2.3-foss-2016a/3.3.1-foss-2016a/g' *.sh
```

**Comment by richelbilderbeek** [comment]
Fixed with a2f335f13afad8539c395b493e07d56355c5b13a
